CS.BL/Services/EmailService.cs

```diff
@@ -100,8 +101,8 @@ public async Task<bool> SendEmail(Ticket ticket)
 
             var context = scope.ServiceProvider.GetRequiredService<ApplicationContext>();
             
-            var admin = await context.Users.FirstOrDefaultAsync(u => u.Id == ticket.AdminId);
-            var user = await context.Users.FirstOrDefaultAsync(u => u.Id == ticket.CustomerId);
+            var admin = await context.Users.FirstAsync(u => u.Id == ticket.AdminId);
```

Comment:
on the left side you do not check if admin or user is null
either add checks, or use FirstAsync and be ready for exceptions